File: clients/javascript/src/models/evictionPolicy.ts
// this file is @generated

export enum EvictionPolicy {
    NoEviction = 'NoEviction',
    LeastRecentlyUsed = 'LeastRecentlyUsed',
    }

export const EvictionPolicySerializer = {
    _fromJsonObject(object: any): EvictionPolicy {
        return object;
    },

    _toJsonObject(self: EvictionPolicy): any {
        return self;
    }
}

File: z-clients/java/src/main/java/com/svix/coyote/models/EvictionPolicy.java
// this file is @generated
package com.svix.coyote.models;

import com.fasterxml.jackson.annotation.JsonValue;

public enum EvictionPolicy {
    NO_EVICTION("NoEviction"),
    LEAST_RECENTLY_USED("LeastRecentlyUsed");
    private final String value;

    EvictionPolicy(String value) {
        this.value = value;
    }

    @JsonValue
    public String getValue() {
        return this.value;
    }
}